firelyserver/releasenotes/releasenotes.rst

```diff
@@ -17,6 +17,19 @@ Upgrading Firely Server
 
 See :ref:`upgrade` for information on how to upgrade to a new version of Firely Server.
 
+.. _vonk_releasenotes_411:
+
+Release 4.1.1 hotfix
+--------------------
+
+Feature
+^^^^^^^
+#. SMART configuration: Some identity providers use multiple endpoints with different base addresses for its authorization operations. Added an extra configuration option ``AdditionalEndpointBaseAddresses`` to define additional authorization base endpoints addresses next to the main authorization endpoint to accomodate this. See :ref:`feature_accesscontrol_config` for further details.
```

Comment:
"... to define additional base endpoints addresses next to the main authority endpoint to accommodate this."